source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
-   rsyslogd during startup processing. Interim changes to the user
-   mapping are *not* detected.
-
-.. function::  fileOwnerNum [uid]
-
-   *Default: system default*
-
-   *Available since: 7.5.8, 8.1.4*
-
-   Set the file owner for files newly created. Please note that this
-   setting does not affect the owner of files already existing. The
-   parameter is a numerical ID, which which is used regardless of
-   whether the user actually exists. This can be useful if the user
-   mapping is not available to rsyslog during startup.
+People often report problems that dynafiles are not properly created.
+Apart from generating invalid paths/filenames, sometimes these problems are due to SELinux rules, which do not permit
+the creation of those files. The same happens for generic permission issues (this is often a problem
+under Ubuntu where permissions are dropped by default)
+
+=========  ==========================
+template:  "template-name"
+Default:   none
+Legacy:    $ActionFileDefaultTemplate
+=========  ==========================
+
+Sets the template to be used for this action.
+
+================  ==============================
+closeTimeout:     "number"
+Default:          "0 if using *file*, 10 if using *dynaFile*"
+Available since:  8.3.3
+================  ==============================
+
+Specifies after how many minutes of inactivity a file is
+automatically closed.
+
+Note that this functionality is implemented based on the  :doc:`janitor process <../../concepts/janitor>`.
+
+==================  ==================
+dynaFileCacheSize:  "number"
+Default:            "10"
+Legacy:             $DynaFileCacheSize
+==================  ==================
+
+**Applies only if dynamic filenames are used.**
+
+Specifies the number of files that will be kept open.
+Note that this is a per-action parameter, so if you have
+multiple dynafile actions, each of them have their individual caches
+(which means the numbers sum up).
+
+Ideally, the cache size exactly
+matches the need. You can use :doc:`impstats <impstats>` to tune
+this value. Note that a too-low cache size can be a very considerable
+performance bottleneck.
+
+=========  ===============
+zipLevel:  "number"
+Default:   "0"
+Legacy:    $OMFileZipLevel
+=========  ===============
+
+If greater than 0, turns on gzip compression of the output file. The
+higher the number, the better the compression, but also the more CPU
+is required for zipping.
+
+================  ========
+veryRobustZip:    "on|off"
+Default:          "on"
+Available since:  7.3.0
+================  ========
+
+If *zipLevel* is greater than 0,
+then this setting controls if extra headers are written to make the
+resulting file extra hardened against malfunction. 
+If set to *off*, data appended to previously unclean closed files may not be
+accessible without extra tools.
+
+**TODO Note that this risk is usually expected to be bearable, and thus "off" is the default mode.**
```

Comment:
If I'm not wrong it's set to 0 (false), but as documentation is inconsistent, I wanted another eyes on it.